backend/.venv/pyvenv.cfg

```diff
@@ -0,0 +1,5 @@
+home = /opt/homebrew/opt/python@3.11/bin
+include-system-site-packages = false
+version = 3.11.6
+executable = /opt/homebrew/Cellar/python@3.11/3.11.6/Frameworks/Python.framework/Versions/3.11/bin/python3.11
+command = /opt/homebrew/opt/python@3.11/bin/python3.11 -m venv /Users/jasmineguruparan/Downloads/CarbonTrack/backend/.venv
```

Comment:
This file should not be commited, please remove and add it to gitignore @Yazan10x

Reply:
Remove the `.venv` folder as a whole and add it to gitignore